config/shared/errors/errors.go

```diff
@@ -55,6 +55,9 @@ var (
 	ErrLuksLabelTooLong          = errors.New("luks device labels cannot be longer than 47 characters")
 	ErrLuksNameContainsSlash     = errors.New("device names cannot contain slashes")
 	ErrInvalidLuksKeyFile        = errors.New("invalid key-file source")
+	ErrUnknownClevisPin          = errors.New("unsupported clevis pin")
+	ErrClevisConfigRequired      = errors.New("clevis config is required when specifying overrides")
```

Comment:
Hmm, maybe we should use `custom` somewhere in there instead of `overrides`? How about just

```suggestion
	ErrClevisConfigRequired      = errors.New("missing required custom clevis config")
```